### DISCUSSION TITLE
"Recent played" beatmap sort/group option

### DISCUSSION THREAD

**Discussion by TheBrunoRM** [ppy/osu#14809] [Ideas] [answered]
This is the sorting/group option that I use the most in stable, would be nice to see it in lazer

**Comment by Totsugeki1** [comment]
Adding to this, maybe another one for maps where the user submitted a score would be useful.

**Comment by peppy** [comment, marked as answer]
It's in stable and will eventually be in lazer.

**Reply by Nichokas** [reply]
Already added to lazer?

**Reply by bdach** [reply]
Sorting yes. Grouping no.
